grammar/cypher.xml

```diff
@@ -196,6 +198,33 @@
     FOREACH &WS; ( &WS; &var; &SP; IN &SP; &expr; &WS; | <repeat min="1">&SP;<non-terminal ref="Clause"/></repeat> &WS; )
   </production>
 
+  <production name="InQueryCall">
+    <seq>CALL &SP; <non-terminal ref="ExplicitProcedureInvocation"/> <opt>&WS; YIELD &SP; <non-terminal ref="YieldItems"/></opt></seq>
+  </production>
+
+  <production name="StandaloneCall">
+    <seq>
+      CALL &SP;
+      <alt>
+        <non-terminal ref="ExplicitProcedureInvocation"/>
+        <non-terminal ref="ImplicitProcedureInvocation"/>
+      </alt>
+      <opt>&SP; YIELD &SP; <non-terminal ref="YieldItems"/></opt>
+    </seq>
+  </production>
+
+  <production name="YieldItems" rr:inline="true">
+    <alt>
+      <seq><non-terminal ref="YieldItem"/> <repeat>&WS; , &WS;<non-terminal ref="YieldItem"/></repeat></seq>
+      -
+    </alt>
+  </production>
+
+  <production name="YieldItem" rr:inline="true">
+    <seq><opt><non-terminal ref="ProcedureResultField"/> &SP; AS &SP;</opt> &var;</seq>
```

Comment:
Yes, the left side is optional. If the variable name is left off, it is taken to be identical to the alias name. This mirrors how things work in our current implementation, we should be able to turn it around if we feel that is more clear.